examples/multigpu/graphbolt/node_classification.py

```diff
@@ -393,6 +404,9 @@ def parse_args():
     # Load and preprocess dataset.
     dataset = gb.BuiltinDataset("ogbn-products").load()
 
+    # Thread limiting to avoid resource competition.
+    os.environ["OMP_NUM_THREADS"] = str(mp.cpu_count() // 2 // world_size)
```

Comment:
torch.set_num_threads(mp.cpu_count() // 2 // world_size)

Reply:
chang back to 
` os.environ["OMP_NUM_THREADS"] = str(mp.cpu_count() // 2 // world_size)`
torch.set_num_threads seems doesn't work before spwan.